src/index.js

```diff
@@ -1,14 +1,34 @@
-import _ from "lodash";
-import "./style.css";
+import './style.css';
 
-function component() {
-  const element = document.createElement("div");
+import Store from './modules/storage.js';
+import User from './modules/Users.js';
+import UI from './modules/UIclass.js';
 
-  // Lodash, currently included via a script, is required for this line to work
-  element.innerHTML = _.join(["Hello", "webpack"], " ");
-  element.classList.add('hello');
+const form = document.querySelector('#form');
 
-  return element;
-}
+// Event : Display UsersScore
+document.addEventListener('DOMcontentLoaded', UI.displayUserScore());
 
-document.body.appendChild(component());
+// Add a user
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  // console.log(e)
+  // get Form values
```

Comment:
- [ ] Please do not leave unused code as a comment in your codebase. unused code as comments makes your code not clean. Kindly remove all unused code this will make your code mush noise, It is always a good practice to keep your code clean from such comments.